### PR TITLE
using django templates on webpages

### LINKED ISSUE #87: Templates should use the extend templates
Please use django template extension, it will reduce the copy of each page with bootstrap and navigation menu, please review GUI guided session

### PR DESCRIPTION
**Pull request by erezdav22**
Added new file- "base_html.html" to the templates file.
This file has the basic HTML code that appears in every page of the site.
All other pages extends this page.
The site is working and everything is right, i just made this a draft PR so i can share the code with you and ask how can i keep the footer at the same point on all pages?
I managed to make that work before i used the extension with the css file, but since i've added the extends- the footer moves to a different spot in each page, this is the only thing i need to fix in order to finish this PR.
thanks!
fixes #87

### CODE REVIEW

File: ultimatejobweb/templates/base_html.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            <li class="page-item">
+                <a href=" /available_jobs" class="page-link">available jobs for you</a>
+            </li>
+        </ul>
+	
+	
+	
+	{% block content %}
+	
+	{% endblock %}
```
Comment (simzacks) [inline ultimatejobweb/templates/base_html.html]: You want to finish the html file here.
You would probably put the footer and then close the body and html tags.

In the other files should only be the stuff that goes inbetween the block content and endblock tags

Comment (Yarboa) [inline ultimatejobweb/templates/base_html.html]: Prefered to close here </body> </html> </header>
Who ever derive from just add the extra classes

File: ultimatejobweb/templates/available_jobs.html [hunk 2]
```diff
@@ -122,4 +77,5 @@ <h1 class="display-3">UltimateJob</h1>
 
 
 </body>
```
Comment (Yarboa) [inline ultimatejobweb/templates/available_jobs.html]: This should move to base_html.html

File: ultimatejobweb/templates/home_page.html [hunk 3]
```diff
@@ -1,54 +1,6 @@
...
-            <li class="page-item">
-                <a href="available_jobs" class="page-link">available jobs for you</a>
-            </li>
-        </ul>
+{% extends 'base_html.html' %}
 
+{% block content %}
     <!--progress bar-->
     <div class="progress">
         <div class="progress-bar progress-bar-striped progress-bar-animated-bg-primary" role="progressbar" aria-valuenow="10" aria-valuemin="0"
```
Comment (Yarboa) [inline ultimatejobweb/templates/home_page.html:6]: Why is it out of block_content?

File: ultimatejobweb/templates/home_page.html [hunk 4]
```diff
@@ -89,4 +41,5 @@ <h1 class="display-3">Welcome to UltimateJob!</h1>
 
 
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (Yarboa) [inline ultimatejobweb/templates/home_page.html]: This should move to base_html.html

File: ultimatejobweb/templates/personal_area.html [hunk 5]
```diff
@@ -83,4 +38,5 @@ <h1 class="display-3">UltimateJob</h1>
 
 
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (Yarboa) [inline ultimatejobweb/templates/personal_area.html]: This should move to base_html.html

File: ultimatejobweb/templates/sign_in.html [hunk 6]
```diff
@@ -87,4 +40,5 @@ <h1 class="display-3">UltimateJob</h1>
 
 
 </body>
```
Comment (Yarboa) [inline ultimatejobweb/templates/sign_in.html]: same here

File: ultimatejobweb/templates/sign_up.html [hunk 7]
```diff
@@ -94,4 +53,5 @@ <h1 class="display-3">UltimateJob</h1>
 
 
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (Yarboa) [inline ultimatejobweb/templates/sign_up.html]: same here

**Comment by erezdav22** [comment]
Just finished to fix all the comments of Sim and Yariv, and now the website is working with base html file, and all the other pages inheriting from the base one.

**Review by simzacks — APPROVED** [review]
LGTM

**Comment by simzacks** [comment]
@Yarboa  the tests that failed have nothing to do with this PR. I say we merge. Do you have a different opinion?

**Comment by Yarboa** [comment]
Looks good to me, nice use of bootstrap and templates extension

**Pull request merged**